Description: Ignore failing tests on ppc64el
 "ignore-powerpc64le" doesn't seem to work when I tested this.
Author: Ximin Luo <infinity0@debian.org>
Bug: https://github.com/rust-lang/rust/issues/44683
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=876134

--- a/src/test/run-pass/smallest-hello-world.rs
+++ b/src/test/run-pass/smallest-hello-world.rs
@@ -10,6 +10,7 @@
 
 // Smallest "hello world" with a libc runtime
 
+// ignore-powerpc
 // ignore-windows
 // ignore-android
 
